test/e2e-common.sh

```diff
@@ -166,6 +175,10 @@ function test_yaml_can_install() {
         for ignore in ${TEST_YAML_IGNORES};do
             [[ ${ignore} == "${testname}" ]] && skipit=True
         done
+
+        # In case if PLATFORM env variable is specified, do the tests only for matching tasks
+        [[ -n ${PLATFORM} ]] && [[ $(grep "tekton.dev/platforms" ${runtest}) != *"${PLATFORM}"* ]]  && skipit=True
```

Comment:
maybe redirect stderr to /dev/null since it may fails (but the case is handled in skipit)

Reply:
@chmouel done